manager.spec.yaml

```diff
@@ -159,6 +159,88 @@ paths:
         - $ref: '#/components/parameters/show_advisories'
         - $ref: '#/components/parameters/advisory'
 
+  /cves/{cve_id}/mitigated_systems:
```

Comment:
the naming might be confusing because the API doesn't cover systems with CVEs mitigated from VMaaS

Reply:
the system might be still vulnerable from VMaaS and it'll be returned in this list

Reply:
@jdobes what name do you suggest?

Reply:
For example `rule_mitigated_systems`.

Reply:
@jdobes 
the system might be still vulnerable from VMaaS and it'll be returned in this list
yes that's the point of rules to tell you that you're not vulnerable by the CVE even if you have outdated package installed (triggers VMaaS hit) but for example you might not have the vulnerable kernel module loaded which makes you not vulnerable. This is the added value of rules.